Mvc.sln

```diff
@@ -123,6 +123,8 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "MvcSandbox", "samples\MvcSa
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "SimpleWebSite", "test\WebSites\SimpleWebSite\SimpleWebSite.xproj", "{396B40D7-AC70-49A7-B33C-ED42129FEBE3}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "DisplayLocalizationSite", "test\WebSites\DisplayLocalizationSite\DisplayLocalizationSite.xproj", "{454C849D-D305-429C-893A-25DCF72CA4A1}"
```

Comment:
Why add another web site? We spent a while _removing_ sites and this should fit in `HtmlGenerationWebSite`.